src/app/docs/utils/fetchMenu.ts

```diff
@@ -0,0 +1,10 @@
+import { Menu } from '../types';
+import { Language } from '@/shared/types';
+
+export const fetchMenu = async (lang: Language): Promise<Menu> => {
+  const data = await fetch(
+    `https://raw.githubusercontent.com/SpaNb4/docs/refs/heads/master/docs/${lang}/docsMenu_${lang}.json`,
```

Comment:
i'd store the first part of the link in a constant since it's repeated too often